vsphere/internal/helper/guestoscustomizations/customizations_helper.go

```diff
@@ -608,12 +610,14 @@ func expandCustomizationGuiUnattended(d *schema.ResourceData, prefix string) typ
 
 // expandCustomizationIdentification reads certain ResourceData keys and
 // returns a CustomizationIdentification.
-func expandCustomizationIdentification(d *schema.ResourceData, prefix string) types.CustomizationIdentification {
+func expandCustomizationIdentification(d *schema.ResourceData, prefix string, version viapi.VSphereVersion) types.CustomizationIdentification {
 	obj := types.CustomizationIdentification{
 		JoinWorkgroup: d.Get(prefix + "workgroup").(string),
 		JoinDomain:    d.Get(prefix + "join_domain").(string),
 		DomainAdmin:   d.Get(prefix + "domain_admin_user").(string),
-		DomainOU:      d.Get(prefix + "domain_ou").(string),
+	}
+	if version.AtLeast(viapi.VSphereVersion{Product: version.Product, Major: 8, Minor: 0, Patch: 2}) {
+		obj.DomainOU = d.Get(prefix + "domain_admin_user").(string)
```

Comment:
`domain_ou`?

Reply:
Sure, it was in a minute after the wrong commit. Could you please check?